lib/app/layouts/conversation_details/widgets/chat_info.dart

```diff
@@ -328,7 +325,7 @@ class _ChatInfoState extends OptimizedState<ChatInfo> {
               child: Row(
                 mainAxisAlignment: kIsWeb || kIsDesktop ? MainAxisAlignment.center : MainAxisAlignment.spaceBetween,
                 children: intersperse(const SizedBox(width: 5), [
-                  if (!kIsWeb && !kIsDesktop && !chat.chatIdentifier!.startsWith("urn:biz")
+                  if (!kIsWeb && !kIsDesktop && !(chat.chatIdentifier?.startsWith("urn:biz") ?? false)
```

Comment:
some backends (mine) don't have chat identifiers

Reply:
How do you tell apart "business" chats? I guess you couldnt really start one from RustPush?